webextension/scripts/utils.js

```diff
@@ -23,6 +19,50 @@ const newshosts = new Set([
   'www.washingtonpost.com'
 ])
 
+/* * * Browser Detection * * */
+
+function getBrowser() {
+  // the order of these is important!
+  if (navigator.brave) { return 'brave' }
+  else if (navigator.userAgent.indexOf('Edg') !== -1) { return 'edge' }
+  else if (navigator.userAgent.indexOf('OPR') !== -1) { return 'opera' }
+  else if (navigator.userAgent.indexOf('Firefox') !== -1) { return 'firefox' }
+  else if (navigator.userAgent.indexOf('Chromium') !== -1) { return 'chromium' }
+  else if (navigator.userAgent.indexOf('Chrome') !== -1) { return 'chrome' }
+  else if (navigator.userAgent.indexOf('Safari') !== -1) { return 'safari' }
+  else if ((navigator.userAgent.indexOf('Trident') !== -1) || (navigator.userAgent.indexOf('MSIE'))) { return 'ie' }
+  else { return '' }
+}
+
+const hostURLs = {
+  chrome: 'https://chrome-api.archive.org/',
+  chromium: 'https://chrome-api.archive.org/',
+  firefox: 'https://firefox-api.archive.org/',
+  safari: 'https://safari-api.archive.org/',
+  brave: 'https://brave-api.archive.org/',
+  edge: 'https://edge-api.archive.org/',
+  ie: 'https://edge-api.archive.org/',
+  opera: 'https://opera-api.archive.org/'
+}
+
+const feedbackURLs = {
+  chrome: 'https://chrome.google.com/webstore/detail/wayback-machine/fpnmgdkabkmnadcjpehmlllkndpkmiak/reviews?hl=en',
+  chromium: 'https://chrome.google.com/webstore/detail/wayback-machine/fpnmgdkabkmnadcjpehmlllkndpkmiak/reviews?hl=en',
+  firefox: 'https://addons.mozilla.org/en-US/firefox/addon/wayback-machine_new/',
+  safari: 'https://apps.apple.com/us/app/wayback-machine/id1201888313'
+}
+
+const gBrowser = getBrowser()
+const isChrome = (gBrowser === 'chrome') || (gBrowser === 'chromium')
```

Comment:
Do we really need these variables (except isFirefox) at this point of time ?

Reply:
I'm going to need them in the future, possibly. I'll have to make some changes like converting all `chrome` to `browser` and there are some browser-specific issues, so to get it all working with Safari as well. So let's keep it for now.

Reply:
Ok. Thank you. !